src/Raven.Server/Documents/PeriodicBackup/Restore/S3RestorePoints.cs

```diff
@@ -69,7 +72,7 @@ protected override ParsedBackupFolderName ParseFolderNameFrom(string path)
         protected override async Task<ZipArchive> GetZipArchive(string filePath)
         {
             var blob = await _client.GetObjectAsync(filePath);
-            var file = await RestoreUtils.CopyRemoteStreamLocallyAsync(blob.Data, _configuration);
+            var file = await RestoreUtils.CopyRemoteStreamLocallyAsync(blob.Data, blob.Size, _configuration, onProgress: null, _cancellationToken);
```

Comment:
progress?

Reply:
Getting the restore points isn't an operation so it doesn't have a progress.